packages/iTwinUI-react/src/core/Buttons/Button/Button.tsx

```diff
@@ -71,14 +71,11 @@ export const Button: ButtonComponent = React.forwardRef((props, ref) => {
   return (
     <Element
       ref={ref}
-      className={cx(
-        'iui-button',
-        `iui-${styleType}`,
-        {
-          [`iui-${size}`]: !!size,
-        },
-        className,
-      )}
+      className={cx('iui-button', className)}
+      data-iui-variant={
+        styleType && styleType !== 'default' ? styleType : undefined
```

Comment:
`styleType` is always defined, so enough
`styleType !== `default` ? styleType : undefined`

Reply:
updated